Tests/LibWeb/AccessibilityBridgeMac/harness/ladybird.py

```
@@ -0,0 +1,282 @@
+        env = os.environ.copy()
+
+        # Send stderr to a regular file rather than subprocess.PIPE: Ladybird debug builds (ENABLE_ALL_THE_DEBUG_MACROS)
+        # produce huge stderr volumes (PROMISE_DEBUG, etc.). A pipe's kernel buffer is ~64 KB; once it fills, every
+        # dbgln in WebContent blocks on write — which can wedge the engine before AX registration completes. A regular
+        # file never backpressures the writer, and we still get the bytes for diagnostics on failure.
+        self._stderr_file = tempfile.NamedTemporaryFile(prefix="ladybird-ax-stderr-", suffix=".log", delete=False)
+
+        # --force-cpu-painting: skip the GPU/Metal path. Per Services/WebContent/main.cpp the CPU backend is what other
+        # test harnesses use ("the GPU backend is not deterministic"), and these AX tests don't depend on rendering at
+        # all — they inspect the NSAccessibility surface, which is independent of which painter ran.
+        self._process = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)